TempleApp/src/main/java/example/TempleApp/Activities/InsertData.java

```diff
@@ -95,16 +95,21 @@ public void onClick(View v) {
             public void onClick(View view) {
                 id = id + uid1ET.getText().toString();
                 name = nameET.getText().toString();
-                if (flag == 1) {
-                    poojaTyp = String.valueOf(poojaType.getSelectedItem());
-
-                    overall = poojaTyp + getResources().getString(R.string.empty) + name + getResources().getString(R.string.empty) + paidCheck;
-                } else {
-                    money = moneyDonated.getText().toString();
-                    overall = money + getResources().getString(R.string.empty) + name + getResources().getString(R.string.empty) + paidCheck;
+                if (id.length() == 3) {
```

Comment:
`id.length() >=3`

Reply:
It won't accept length > 3, please change than.

Reply:
no @yashk2000 ...actual length of id is 3..i.e if nothing is entered..id length will be 3,then it wont allows the user...so if anything is entered,it will be greater than 3 and it allows the user.

Reply:
In that case it's fine :+1: